legend-engine-xt-relationalStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/result/RealizedRelationalResult.java

```diff
@@ -82,6 +83,15 @@ public RealizedRelationalResult(RelationalResult relationalResult) throws SQLExc
         }
     }
 
+    public int getRowLimit()
+    {
+        if (System.getProperty(ROW_LIMIT_PROPERTY_NAME) == null)
+        {
+            return DEFAULT_ROW_LIMIT;
+        }
+        return Integer.valueOf(System.getProperty(ROW_LIMIT_PROPERTY_NAME));
```

Comment:
Lets simplify this with a call to [getInteger](https://docs.oracle.com/javase/8/docs/api/java/lang/Integer.html#getInteger-java.lang.String-java.lang.Integer-)

```java
Integer.getInteger(ROW_LIMIT_PROPERTY_NAME, DEFAULT_ROW_LIMIT)
```

This will return the default if property is not present, otherwise will get it and parse it.